content/version/1/4/code-of-conduct.gu.md

```diff
@@ -0,0 +1,76 @@
++++
+version = "1.4"
+aliases = ["/version/1/4/gu"]
++++
+
+# ફાળકોના ઠરાવની આચારસંહિતા
+
+## અમારી પ્રતિજ્ઞા
+
+ખુલ્લા અને આવકાર્ય વાતાવરણ ઉત્તેજીત કરવાનાં હિતમાં, અમે ફાળકો અને જાળવણારાઓ,
+પ્રતિજ્ઞા લઈએ છીએ કે અમારા પ્રકલ્પમાં અને સમુદાયમાં સહભાગ આ બધાં માટે, ઉંમર,
+શરીરાકાર, અપંગત્વ, વંશ, લૈંગિક વૈશિષ્ટ્ય, લૈંગિક અવતાર, અનુભવ, ભણતર,
+આર્થિક સ્થિતિ, રાષ્ટ્રીયતા, વ્યક્તિગત અવતાર, જાત, પન્થ અથવા કામસિક પસન્દ, આની
+પર્વા કર્યા વગર, એક છળમુક્ત અનુભવ રહેશે.
+
+## અમારા ધોરણો
+
+શામેલ વર્તનનાં ઉદાહરણો જે સકારાત્મક વાતાવરણ બનાવવામાં મદદગાર છે:
+
+* આવકારક અને સમાવિષ્ટ ભાષાનો ઉપયોગ
+* જુદા જુદા દ્રષ્ટિકોણો અને અનુભવોનું માન રાખવું
+* રચનાત્મક ટીકા કૃપાથી સ્વીકારવું
```

Comment:
રચનાત્મક ટીકા `કૃપાથી` સ્વીકારવું -> કૃપાથી maybe is the literal translation of graceful(that doesn't fit well) maybe we can use  `ખુશી ખુશી` ( I am not sure but `ખેલદિલી` can also be used in some cases )